engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/runner/FlinkStubbedRunner.scala

```diff
@@ -37,19 +37,25 @@ trait FlinkStubbedRunner {
     val configuration: Configuration = new Configuration
     configuration.addAll(jobGraph.getJobConfiguration)
     configuration.setInteger(TaskManagerOptions.NUM_TASK_SLOTS, env.getParallelism)
+
+    configuration.setInteger(JobManagerOptions.PORT, 0)
```

Comment:
Why it need to be set? What will happen after that?

Reply:
good question... I removed it and tests still pass...